docs/changelog.txt

```diff
@@ -39,6 +39,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Removed
 
 ## Fixes
+- `cxxrandom`: fixed problems with interface and updated docs
```

Comment:
since this change is specific to `bool_distribution`, it would be more useful to readers if we name it directly here. also, the changelog entry should describe exactly what was fixed. For example:
```
- `cxxrandom`: fixed exception when calling ``bool_distribution``
```
and unless the PR contains significant additions to documentation (like entire new sections or examples), we don't need to mention doc updates here

Reply:
So I guess I should probably add something about the usage examples I added.

Reply:
👍